src/Apps/ArtAdvisor/07-Curated-Discovery/Components/Result/Artwork.tsx

```diff
@@ -74,22 +78,67 @@ export const Artwork: FC<ArtworkProps> = props => {
     }
   }
 
+  const resizedImage = resize(
+    artwork.imageUrl || "https://www.artsy.net/images/missing_image.png",
+    { height: IMAGE_HEIGHT }
+  )
+
   return (
     <Box>
-      <Image src={artwork.imageUrl} height={300} />
-      <Flex py={1} gap={1} justifyContent={"center"}>
-        <Clickable onClick={handleClickLike}>
-          <CheckmarkFillIcon fill={"black60"} size={30} />
-        </Clickable>
-        <Clickable onClick={handleClickDislike}>
-          <CloseFillIcon fill={"black60"} size={30} />
-        </Clickable>
-      </Flex>
-      <Flex flexDirection="column" alignItems="center">
-        <a href={`/artwork/${artwork.slug}`} target="_blank">
-          {artwork.title}
-        </a>
-        <Text>{artwork.price}</Text>
+      <RouterLink
+        to={`/artwork/${artwork.slug}`}
+        target="_blank"
+        textDecoration={"none"}
+        overflow={"hidden"}
+      >
+        <Image src={resizedImage} height={IMAGE_HEIGHT} />
+      </RouterLink>
+
+      <Flex
+        alignItems={"flex-start"}
+        justifyContent={"space-between"}
+        gap={1}
+        pt={0.5}
+      >
+        <RouterLink
+          to={`/artwork/${artwork.slug}`}
+          target="_blank"
+          textDecoration={"none"}
+          overflow={"hidden"}
+        >
```

Comment:
@mc-jones Good point about the new window, esp since we'd lose state by navigating away. I forgot that I did actually sort that out for the artwork cards, with the `target` attribute here on L105. I'll copy that over to the other two cards as well.